README.md

```diff
@@ -1 +1,108 @@
 #Session Service
+## Overview
+The `TDS_ConfigService` (aka Config Support Service) consists of two modules:
+
+* **client:** Contains the POJOs/classes needed for a consumer to interact with the Session Support Service
+* **service:** REST endpoints that provide TDS session data
+
+## Build
+To build the **client**:
+
+* `mvn clean install -f /path/to/client/pom.xml`
+
+To build the **service**:
+
+* `mvn clean install -f /path/to/service/pom.xml`
+
```

Comment:
Should we just put one command for building them both?  `mvn clean install -f /path/to/parent/pom.xml`